server/middleware/validateParty.js

```diff
@@ -101,12 +101,10 @@ class ValidateParties {
 
     if (!validate.logoUrl.test(logoUrl)) {
       error = 'Invalid party logo';
-    }
-    if (!logoUrl || logoUrl === undefined) {
+    } else if (!logoUrl || logoUrl === undefined) {
       error = 'Party  must be specified';
```

Comment:
I think this should be Party logo must be specified instead.

Reply:
Acknowledged and resolved